package.json

```diff
@@ -305,7 +305,7 @@
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
     "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#3a0946727d9013ff2771af37bfca0c4c0ff0a1d5",
-    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.11.0",
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library/releases/download/wc-v0.11.0/web-components-v0.10.0.tgz",
```

Comment:
The numbers are different because `wc-v0.11.0` is the _release_, and the part of the filename after the `-` is based on the version in `package.json`, which I didn't update :upside_down_face: